.github/workflows/release.yml

```diff
@@ -89,6 +89,15 @@ jobs:
         if: matrix.build == 'macos'
         run: brew install p7zip
 
+      - name: Install OpenSSL via vcpkg (Windows)
+        if: matrix.build == 'windows'
+        run: |
+          git clone https://github.com/microsoft/vcpkg
+          .\vcpkg\bootstrap-vcpkg.bat
+          .\vcpkg\vcpkg install openssl:x64-windows
+        env:
+            VCPKGRS_DYNAMIC: 1
+
```

Comment:
mmmm - if we need an extra install step for windows then i dont think we should use openssl crate to do this. generally we want wrangler to be a standalone binary that does not require extra system configuration